sdk/all/src/main/java/io/opentelemetry/sdk/OpenTelemetrySdk.java

```diff
@@ -216,10 +216,6 @@ public OpenTelemetrySdk build() {
       MeterProvider meterProvider = buildMeterProvider();
       TracerSdkProvider tracerProvider = buildTracerProvider();
 
-      for (SpanProcessor spanProcessor : spanProcessors) {
-        tracerProvider.addSpanProcessor(spanProcessor);
-      }
```

Comment:
the downside about removing this is that now we won't be adding span processors to a pre-configured SDK. Not sure if we're ok with that or not. I know we've chatted about removing the ability to provide the fully configured SDK altogether, which would make this (obviously) just fine.